lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -0,0 +1,417 @@
+    ).tag(config=True)
+
+    point_like = traits.Bool(
+        help="True for point-like IRF, False for Full Enclosure",
+        default_value=False,
+    ).tag(config=True)
+
+    overwrite = traits.Bool(
+        help="If True, overwrites existing output file without asking",
+        default_value=True,
+    ).tag(config=True)
+
+    classes = [DataSelection, DataBinning]
+
+    aliases = {
+        ("fg", "input_gamma_dl2"): "IRFFITSWriter.input_gamma_dl2",
+        ("fp", "input_proton_dl2"): "IRFFITSWriter.input_proton_dl2",
+        ("fe", "input_electron_dl2"): "IRFFITSWriter.input_electron_dl2",
+        ("o", "output_irf_file"): "IRFFITSWriter.output_irf_file",
+        ("pnt", "point_like"): "IRFFITSWriter.point_like",
+        ("int", "intensity"): "DataSelection.intensity",
+        ("len", "length"): "DataSelection.length",
+        ("w", "width"): "DataSelection.width",
+        "r": "DataSelection.r",
+        "wl": "DataSelection.wl",
+        ("leak_2", "leakage_intensity_width_2"):
+            "DataSelection.leakage_intensity_width_2",
+        ("gh", "fixed_gh_cut"): "DataSelection.fixed_gh_cut",
+        ("theta", "fixed_theta_cut"): "DataSelection.fixed_theta_cut",
+        ("src_fov", "fixed_source_fov_offset_cut"):
+            "DataSelection.fixed_source_fov_offset_cut",
+        "lst_tel_ids": "DataSelection.lst_tel_ids",
+        ("etrue", "true_energy_bins"): "DataBinning.true_energy_bins",
+        ("ereco", "reco_energy_bins"): "DataBinning.reco_energy_bins",
+        ("emigra", "energy_migra_bins"): "DataBinning.energy_migra_bins",
+        ("sing_fov", "single_fov_offset_bins"): "DataBinning.single_fov_offset_bins",
+        ("mult_fov", "multiple_fov_offset_bins"): "DataBinning.multiple_fov_offset_bins",
+        ("bkg_fov", "bkg_fov_offset_bins"): "DataBinning.bkg_fov_offset_bins",
+        ("src_off", "source_offset_bins"): "DataBinning.source_offset_bins",
+        "overwrite": "IRFFITSWriter.overwrite",
+    }
+
+    flag = {
+        "point_like": (
+            {"IRFFITSWriter": {"point_like": False}},
+            "Full Enclosure IRFs will be produced",
+        ),
+        "overwrite": (
+            {"IRFFITSWriter": {"overwrite": True}},
+            "overwrites output file",
+        )
+    }
+
+    def __init__(self, config=None, **kwargs):
+        super().__init__(config=config, **kwargs)
+
+    def setup(self):
+
+        if self.output_irf_file.absolute().exists():
+            if self.overwrite:
+                self.log.warning(f"Overwriting {self.output_irf_file}")
+                self.output_irf_file.unlink()
+            else:
+                raise ToolConfigurationError(
+                    f"Output file {self.output_irf_file} already exists,"
+                    " use --overwrite to overwrite"
+                )
+                sys.exit(1)
+        filename = self.output_irf_file.name
+        if filename.split(".")[1:] != ["fits", "gz"]:
+            self.log.debug(
```

Comment:
Please don't do this! If the user specifies `.fits` they probably want `.fits`. And astropy supports it just fine.

Also, overwriting user input should not be `DEBUG`, at least it should be warning. But gain, no reason to force something on the user here.

Reply:
Ok. I will amend the condition to check both `.fits` and `.fits.gz` and change the log type